.github/workflows/develocity-verify-tos.yml

```diff
@@ -0,0 +1,32 @@
+name: Develocity - Terms of Service acceptance verification
+
+on:
+  # issue_comment event is triggered when a pull-request is commented
+  issue_comment:
+    types: [ created ]
+  pull_request_target:
+
+jobs:
+  run-terms-of-service-acceptance:
+    if: github.repository == 'quarkusio/quarkus'
+    runs-on: ubuntu-latest
+    permissions:
+      # required to update signature file
+      contents: write
+      # required to comment pull-request
+      pull-requests: write
+      # required to update pull-request status check
+      actions: write
+      statuses: write
+    steps:
+      - name: Run Terms of Service acceptance verification
+        uses: gradle/github-actions/terms-of-service-acceptance/run@v1-beta
+        with:
+          # tos-location can also point to a file in a Github repository with this syntax: /<owner>/<repo>/blob/<branch>/tos.html
+          tos-location: 'https://foo.bar/tos.html'
+          # Optional inputs
+          pr-comment-tos-acceptance-missing: 'Quarkus uses Develocity to provide insights about CI builds. Please accept [Develocity Terms Of Service]({0}) to get your pull request Build Scan published by commenting this pull-request with the following message:'
```

Comment:
these does not read "right" to me.

I don't have a better proposal ATM but

1) can we somehow explain/express why we need their accept to publish something they already submitted?

2) can we explain/express what happens if they do not and how it is fine if they don't?